modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm

```diff
@@ -45,7 +45,9 @@ sub default_options {
         geneset_types        => [], # Possible values: 'Geneset_EMBL', 'Geneset_FASTA', 'Geneset_GFF3', 'Geneset_GFF3_ENA', 'Geneset_GTF', 'Xref_TSV'
         rnaseq_types         => [], # Possible values: 'RNASeq_Exists'
 
-        dump_metadata        => 0,
+    	homology_dumps       => [], # Possible values : 'Homologies'
```

Comment:
For consistency, I'd rather name this var `homology_types`, so the day we have another dumps format, say newick or others. And the possible values being `Homology_TSV` (then could move to `Homology_Newick` etc...

Reply:
```suggestion
    	homology_types       => [], # Possible values : 'Homology_TSV'
```